health/memoryRecord.go

```diff
@@ -0,0 +1,69 @@
+package health
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"runtime"
+	"runtime/pprof"
+	"strings"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+)
+
+var _ record = (*memoryRecord)(nil)
+
+type memoryRecord struct {
```

Comment:
the name is a bit misleading. "Memory" took me thinking that the record would have been kept only in memory. Surprisingly I have found out that it was persisted in a file.

Reply:
Renamed to `memoryUsageRecord`.